projects/demo/src/modules/components/loader/loader.component.ts

```diff
@@ -10,6 +10,9 @@ import {default as example2Html} from '!!raw-loader!./examples/2/index.html';
 import {default as example2Ts} from '!!raw-loader!./examples/2/index.ts';
 import {default as example3Html} from '!!raw-loader!./examples/3/index.html';
 import {default as example3Ts} from '!!raw-loader!./examples/3/index.ts';
+import {default as example4Html} from '!!raw-loader!./examples/4/index.html';
+import {default as example4Ts} from '!!raw-loader!./examples/4/index.ts';
+import {default as exampleOptions} from '!!raw-loader!./examples/import/define-options.txt';
 import {default as exampleModule} from '!!raw-loader!./examples/import/import-module.txt';
 import {default as exampleHtml} from '!!raw-loader!./examples/import/insert-template.txt';
```

Comment:
Please, refactor this

<img width="569" alt="image" src="https://user-images.githubusercontent.com/12021443/155282773-7a9c46cd-edf5-427f-b834-af5824098b9e.png">

Reply:
done ✅